internal/discover/discover.go

```diff
@@ -86,17 +86,10 @@ func (d *Discoverer) StartLocal(localPort int, localMCAddr string) {
 }
 
 func (d *Discoverer) startLocalIPv4Broadcasts(localPort int) {
-	bb, err := beacon.NewBroadcast(localPort)
-	if err != nil {
-		if debug {
-			l.Debugln("discover: Start local v4:", err)
-		}
-		l.Infoln("Local discovery over IPv4 unavailable")
```

Comment:
Added that back as the Warnln for when opening/binding the socket fails. The the most common case that would be fatal... I can't think of a reason that both opening the listening socket and a random-port sending socket would fail, but if it does we get a duplicate... No big deal I think.